src/dotnet/APIView/APIViewWeb/Client/src/pages/review.ts

```diff
@@ -193,14 +193,18 @@ $(() => {
   $("#reviewSubscribeSwitch").on('change', function () {
     $("#reviewSubscribeForm").submit();
   });
+  // Toggle Viewed Switch
+  $("#reviewViewedSwitch").on('change', function () {
+    $("#reviewViewedForm").submit();
+  });
   // Toggle Close Switch
   $("#reviewCloseSwitch").on('change', function () {
     $("#reviewCloseForm").submit();
   });
 
   // Manage Expand / Collapse State of options
   [$("#approveCollapse"), $("#requestReviewersCollapse"), $("#reviewOptionsCollapse"), $("#pageSettingsCollapse"),
-    $("#associatedPRCollapse"), $("#associatedReviewsCollapse"), $("#generateAIReviewCollapse")].forEach(function (value, index) {
+    $("#associatedPRCollapse"), $("#associatedReviewsCollapse"), $("#generateAIReviewCollapse"), $("#apiRevisionOptionsCollapse")].forEach(function (value, index) {
```

Comment:
Cookie name used here should match everywhere else. Cookie name is case sensitive.